src/main/java/io/github/christophermanahan/carnitas/Main.java

```diff
@@ -14,9 +14,7 @@ public static void main(String[] args) {
         ) {
             connectionAcceptor = acceptor;
             new HTTPServer(acceptor, new ErrorLogger()).run();
-        } catch (IOException e) {
-            throw new RuntimeException(e);
-        }
+        } catch (IOException ignored) {}
```

Comment:
I think it is probably still a good idea to log that something happened. I just don't think we need to rethrow a runtime exception.

Reply:
Logged.